test/unit/tests/002recordExpressionParser.test.js

```diff
@@ -168,8 +166,8 @@ describe('RecordExpressionParser Test', () => {
     // global objects (String)
     { q: 'String.fromCharCode(65, 66, 67)', r: 'ABC' },
     // global objects (unknown objects/functions)
-    { q: 'Invalid.func(1)', e: new SystemError(Validation.messageKeys.expressions.unableToFindNode) },
-    { q: 'Math.unknownFunc(1)', e: new SystemError('undefinedFunction') },
+    { q: 'Invalid.func(1)', e: new SystemError('expression.identifierNotFound') },
+    { q: 'Math.unknownFunc(1)', e: new SystemError('expression.identifierNotFound') },
```

Comment:
should there be some kind of message keys object for these keys? should it go to arena-core?

Reply:
yes we should add this

Reply:
ok... should it have nested objects? like: 
systemErrorKeys: {
  expression: {
    identifierNotFound
  }
}
or should it be just an enum?
SystemErrorKeys {
  ExpressionIdentifierNotFound,
  ...
}